python/pyspark/ml/tests.py

```diff
@@ -832,6 +860,96 @@ def test_logistic_regression(self):
         except OSError:
             pass
 
+    def logistic_regression_check_thresholds(self):
+        self.assertIsInstance(
+            LogisticRegression(threshold=0.5, thresholds=[0.5, 0.5]),
+            LogisticRegressionModel
+        )
+
+        self.assertRaisesRegexp(
+            ValueError,
+            "Logistic Regression getThreshold found inconsistent.*$",
+            LogisticRegression, threshold=0.42, thresholds=[0.5, 0.5]
+        )
+
+    def test_binomial_logistic_regression_bounds(self):
```

Comment:
For PySpark, we should only check the output is consistent with Scala. The most straight-forward way for this test should be loading data directly and run constraint LR on it:
```
data_path = "data/mllib/sample_multiclass_classification_data.txt"
df = spark.read.format("libsvm").load(data_path)
......
```
This will make the test case simple and time-saving. Thanks.